apps/full-site-editing/full-site-editing-plugin/full-site-editing/utils/class-a8c-wp-template.php

```diff
@@ -88,7 +123,13 @@ public function get_header_id() {
 			return null;
 		}
 
-		return $template_blocks[0]['attrs']['templateId'];
+		$header_id = $template_blocks[0]['attrs']['templateId'];
```

Comment:
Is this camelCase because of something happening client-side?..

Reply:
Yes, that's the name of the Template Part block attribute that references it's post, and since it was defined in block's JS it uses the camelCase.